translations/de/info/task_description.html

```diff
@@ -21,11 +21,10 @@
     Eine Schachpartie wird auf einem quadratischen Brett gespielt, das in 8 x 8 abwechselnd
     schwarze und weiße quadratische Felder eingeteilt ist.
     Die Zeilen werden als Reihen bezeichnet und mit den Zahlen 1 bis 8 beschschriftet,
-    während die Spalten als Linien bezeichnet und mit den Buchstaben a bis h beschschriftet
+    während die Spalten als Linien bezeichnet und mit den Buchstaben a bis h beschriftet
     werden.
     So lässt sich jedem Feld ein eindeutiges Koordinatenpaar zuordnen - ein Buchstabe und
-    eine Zahl.
-    (z.B.: "a1", "h8", "d6")
+    eine Zahl. (z.B.: "a1", "h8", "d6")
 </p>
```

Comment:
....eine Zahl (z.B.: "a1", "h8", "d6"). <-- period at the end here, the parantheses.